ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/representative-journey/pages/about-your-computer/about-your-computer.component.spec.ts

```diff
@@ -1,8 +1,22 @@
 import { AboutYourComputerComponent } from './about-your-computer.component';
 import { CanCreateComponent } from '../representative-base-component/component-test-bed.spec';
+import { CannotProceeedUntilChoiceIsSelected } from '../../components/suitability-choice-page-base.component.spec';
 
 describe('AboutYourComputerComponent', () => {
   it('can be created', () => {
     CanCreateComponent(AboutYourComputerComponent);
   });
+  it('cannot proceed to next step until pressing choice, after submit value is bound', () => {
+    const fixture = CannotProceeedUntilChoiceIsSelected(AboutYourComputerComponent);
+
+    // and value is bound
+    expect(fixture.component.model.camera).toBe(0);
+  });
+  it('should initialize choice value', () => {
+    const fixture = CannotProceeedUntilChoiceIsSelected(AboutYourComputerComponent);
```

Comment:
the `CannotProceeedUntilChoiceIsSelected` is actually already calling `ngOnInit`.

Maybe you can run this test only on the component instance instead of using the test bed (makes it a bit faster to run)?

Reply:
removed